src/test/java/org/programmers/springbootbasic/voucher/VoucherServiceTest.java

```diff
@@ -0,0 +1,92 @@
+package org.programmers.springbootbasic.voucher;
+
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+import org.mockito.InjectMocks;
+import org.mockito.Mock;
+import org.programmers.springbootbasic.voucher.model.FixedAmountVoucher;
+import org.programmers.springbootbasic.voucher.model.PercentDiscountVoucher;
+import org.programmers.springbootbasic.voucher.model.Voucher;
+import org.programmers.springbootbasic.voucher.model.VoucherType;
+import org.programmers.springbootbasic.voucher.repository.VoucherRepository;
+import org.programmers.springbootbasic.voucher.service.VoucherService;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Optional;
+import java.util.UUID;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.assertj.core.api.Assertions.assertThatThrownBy;
+import static org.mockito.Mockito.*;
+
+class VoucherServiceTest {
+
+    @Mock
+    VoucherRepository voucherRepository = mock(VoucherRepository.class);
+
+    @InjectMocks
+    VoucherService voucherService = new VoucherService(voucherRepository);
```

Comment:
```suggestion
    @InjectMocks
    private VoucherService voucherService;
```

작성하신 방법처럼 초기화를 하는 경우, @InjectMocks 가 필요 없고, 
Mockito를 활용하려는 경우에는 클래스 위에 @ExtendWith(MockitoExtension.class)를 붙이시면 됩니다.